dockerfiles/theia/src/extensions.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
what is doing this file ? (in comparison with npmjs where some extensions are deployed ? )

Reply:
It's file with list default extensions

Reply:
my question is: what's the difference with https://github.com/eclipse/che/blob/db0cb63926b0110e05d4ba6d78912f1f4d856f95/dockerfiles/theia/src/package.json#L14 ?

Reply:
I didn't saw this changes, I will check it. add-extension.js use extensions.js script to get default extensions from github or volume, build these extensions and rebuild theia with them. Do you propose doesn't support this stuff?

Reply:
I was suggesting as higlighted by @sunix that if extension is stored on npmjs then we can use npmjs way of grabbing extension.

Reply:
@benoitf I checked factory extension works with this pr too. I think, if we have released extension we can apply it to the package.json, like it was done by sunix. But in case if we have github link or volume to the source code we can use add-extensions.js script to development purpose. add-extensions.js contains optimization to reuse Theia node_modules. @benoitf  What do you think? Like variant I can apply empty extensions.json, like sample. And we will apply machine-extension and che-theia-hosted-plugin-manager-extension later after release on the npm.

Reply:
@AndrienkoAleksandr yes it's nice to keep extensibility with this file but  if possible use package.json for released artifacts on npmjs